cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/SoleTrader.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.time.LocalDate'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)